cms/templatetags/cms_tags.py

```diff
@@ -184,11 +183,12 @@ def _get_placeholder(current_page, page, context, name):
     placeholders = page.rescan_placeholders().values()
     fetch_placeholders = []
     request = context['request']
+    site_id = get_site_id(getattr(page, 'site_id', None))
```

Comment:
why ``getattr``? Page objects always have a not nullable site field so you can just use page.site_id